web/templates/user_profile_modal.hbs

```diff
@@ -58,31 +58,50 @@
                                     </div>
                                     {{/if}}
                                 </div>
+                                <div id="content">
```

Comment:
This refactoring also solves a bug.

Bug (before) - 

On having very long links in custom profile fields link, notice the horizontal scroll bar.

![Screenshot_20230420_114526](https://user-images.githubusercontent.com/96468766/233274765-30373c87-aa13-4c9d-abbf-e3f51ec35ac3.png)

![Screenshot_20230420_114531](https://user-images.githubusercontent.com/96468766/233274775-2250dfd0-754c-4b57-960d-3fc532a63cf9.png)

Fix (After) - 

No horizontal scroll bar.

![Screenshot_20230420_104215](https://user-images.githubusercontent.com/96468766/233275034-8887f8f7-fa03-46d8-badc-0e64eac5c717.png)

Reply:
The same bug on small devices too.

<details>

<summary>Bug (before)</summary>

![Screenshot_20230420_114941](https://user-images.githubusercontent.com/96468766/233277157-6cf2560c-25e3-4974-9e6d-4211a3201238.png)

![Screenshot_20230420_114945](https://user-images.githubusercontent.com/96468766/233277164-b8ad332e-5a2c-4a02-9e57-3cd4a28e5b26.png)

</details>

<details>

<summary>Fix (After)</summary>

![Screenshot_20230420_115833](https://user-images.githubusercontent.com/96468766/233277166-0a630753-8426-4906-a0d4-6cac130692f0.png)

</details>

Reply:
This was actually implemented to have the custom profile fields start below the avatar and have full space. There is some discussion in #22202 and [here](https://chat.zulip.org/#narrow/stream/101-design/topic/Full.20Profile.20Layout.20.2321805).
But now after adding those links below the avatar, we might not be able to have full space below the avatar available for custom profile fields, but just wanted to point it out in case we want to have more discussion. @alya FYI.

Anyways, if we move forward with this change, I think it would be better to have it in a separate commit.

Reply:
> But now after adding those links below the avatar, we might not be able to have full space below the avatar available for custom profile fields

Yes. This is happening if we still use the concept of `bottom` for `custom profile fields`.
 
![Screenshot_20230501_143945](https://user-images.githubusercontent.com/96468766/235435838-5f242e5c-59d9-4ab8-97b7-7f679100e548.png)

> Anyways, if we move forward with this change, I think it would be better to have it in a separate commit.

I've made a separate commit for this change.

Reply:
I will wait for feedback from Alya here without doing any further change. 

Also, you would need to update the commit message since this is not a refactor, you are actually changing the html structure to fix or change the existing behavior.

Reply:
> I will wait for feedback from Alya before without doing any further change.

Alright.

> Also, you would need to update the commit message since this is not a refactor, you are actually changing the html structure to fix or change the existing behavior.

Yea sorry for that. I've changed the commit message.